config/shibboleth.yml

```diff
@@ -0,0 +1,6 @@
+development:
+  shibboleth_enabled: <%= ENV['SHIBBOLETH_SSO_ENABLED'] || 'false' %>
```

Comment:
Rails doesn't care what it's called - this isn't actually doing any authentication.  It's only being used to define the global variable "SHIBBOLETH_ENABLED" for us to use to handle two versions of the code - shibboleth and not-shibboleth. "Authentication" seemed too broad to me, since the value contained is whether or not shibboleth is enabled.